Do not generate suffixed cookies (e.g. "UMCSessionId-$PORT"),
because the portal-frontend does not handle them correctly.

--- /usr/lib/python3/dist-packages/univention/management/console/resource.py
+++ /usr/lib/python3/dist-packages/univention/management/console/resource.py
@@ -185,13 +185,7 @@
             return morsel.value

     def suffixed_cookie_name(self, name):
-        host, _, port = self.request.headers.get('Host', '').partition(':')
-        if port:
-            try:
-                port = '-%d' % (int(port),)
-            except ValueError:
-                port = ''
-        return '%s%s' % (name, port)
+        return name

     def bind_session_to_ip(self):
         ip = self.get_ip_address()
